### PR TITLE
Validator Maintenance Page

### PR DESCRIPTION
**Pull request by jimmy3dita**
@bowenwang1996 please review maintenance.md as I went freestyle on the planned maintenance explanation.
@AnaisUrlichs I've updated the community channels, links work, need to be sure we are on the same page.

Checked locally the new menu item, didn't encounter any issue.
@amgando Travis CI fails on `workshop` links.

### CODE REVIEW

File: docs/validator/validator-faq.md [hunk 1]
```diff
@@ -19,6 +19,13 @@ More specific steps:
 3. Send a staking transaction using your wallet / CLI with your account including amount and public key from newly generated key pair.
 4. Wait until the node becomes a validator
 
+<blockquote class="warning">
+<strong>heads up</strong><br><br>
+
+External validators can't join MainNet or TestNet, they can only test their nodes on BetaNet. More info below
```
Comment (bowenwang1996) [inline docs/validator/validator-faq.md:25]: They can join testnet but cannot validate.

File: docs/validator/staking-overview.md [hunk 2]
```diff
@@ -27,4 +27,5 @@ The protocol automatically elects validators by issuing an auction for each epoc
     * [Economics in Sharded Blockchain](https://nearprotocol.com/papers/economics-in-sharded-blockchain/) to know more about the incentives structure of NEAR.
     * [Sharding Design: Nightshade](https://nearprotocol.com/papers/nightshade) to gain a more detailed understanding of our consensus mechanism.
 
-4. If anything is unclear, you get stuck, or you just want to chat with fellow validators, please head over to our [Telegram group](https://t.me/near_validators) or [Discord](https://discord.gg/ZMPr3VB). There we have a dedicated validator channel to answer all of your questions.
+4. If anything is unclear or you get stuck, please head over to our official chat on [Slack](https://near.chat) in the channels `#community-validator-announcement` and `#community-validator-troubleshooting`.
```
Comment (AnaisUrlichs) [inline docs/validator/staking-overview.md:30]: We are likely going to connect our Slack channel with the Telegram and Discord. Thus, please leave those links and do not link to the Slack. I will keep you informed should anything change.

**Comment by amgando** [comment]
> _@amgando Travis CI fails on workshop links_

@jimmy3dita you can ignore this, np.  thanks for checking what was breaking CI

File: docs/validator/keys.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ There are 3 types of key pairs on the NEAR platform
 - validator keys
 - node keys
 
-**Signer keys** are the ones we all know and love.  They're used by all accounts on the network to sign transactions like `sendMoney` and `stake` before sending them to the network.  Signer keys are not related to running a node in any way.  End users who sign up through the [NEAR Wallet](https://wallet.testnet.nearprotocol.com/) get their own signer keys, for example. These are the keys that humans think about and keep safe.
+**Signer keys** are the ones we all know and love.  They're used by all accounts on the network to sign transactions like `sendMoney` and `stake` before sending them to the network.  Signer keys are not related to running a node in any way.  End users who sign up through the [NEAR Wallet](https://wallet.betanet.nearprotocol.com/) get their own signer keys, for example. These are the keys that humans think about and keep safe.
```
Comment (amgando) [inline docs/validator/keys.md:19]: please update this URL, we just merged a PR that moves `nearprotocol.com` to `near.org`

File: docs/validator/keys.md [hunk 4]
```diff
@@ -54,6 +54,10 @@ If validator keys are changed _during an epoch in which the validator is staking
 
 For concrete examples of keys being used as identifiers, you can see a list of validators and active nodes on various NEAR networks here:
 
-- NEAR TestNet
+- NEAR TestNet (staking currently disabled)
   - https://rpc.testnet.nearprotocol.com/status
   - https://rpc.testnet.nearprotocol.com/network_info
+
+- NEAR BetaNet
```
Comment (amgando) [inline docs/validator/keys.md:61]: please update these URL, we just merged a PR that moves `nearprotocol.com` to `near.org`

File: docs/validator/keys.md [hunk 5]
```diff
@@ -54,6 +54,10 @@ If validator keys are changed _during an epoch in which the validator is staking
 
 For concrete examples of keys being used as identifiers, you can see a list of validators and active nodes on various NEAR networks here:
 
-- NEAR TestNet
+- NEAR TestNet (staking currently disabled)
```
Comment (amgando) [inline docs/validator/keys.md:57]: please update these URL, we just merged a PR that moves `nearprotocol.com` to `near.org`

File: docs/validator/validator-faq.md [hunk 6]
```diff
@@ -47,7 +54,8 @@ No. However, the protocol measures the uptime of each validator, and if the gene
 
 ### What are the responsibilities of a validator?
 
-High level, validators must run node and be mostly online. However, it's very important to keep private keys safe, otherwise adversaries might use them to sign malicious blocks, and trigger the protocol slashing.
+High level, validators must run node and be mostly online. Also, they have to be constantly connected on the official [Slack chat](https://near.chat) in the `#community-validator-announcement` channel, in case of emergencies and upcoming hard forks.
```
Comment (bowenwang1996) [inline docs/validator/validator-faq.md:57]: Do we want to switch to discord instead?

File: docs/validator/validator-faq.md [hunk 7]
```diff
@@ -60,7 +68,7 @@ Follow [this tutorial.](local-setup/running-testnet.md)
 
 ### Do validators receive incentives for testing the protocol?
 
-We don’t offer rewards to validators at this point in time. However, we may offer bounties for reporting critical bugs or valuable contributions to the codebase on [GitHub](https://github.com/nearprotocol/). Just keep an eye for all the “good first issue” posts. In the meantime, join the [Telegram Validator Channel](https://t.me/near_validators) to be constantly updated, and be the first to know if we plan to offer incentives in the future. 
+We don’t offer rewards to validators at this point in time. However, we may offer bounties for reporting critical bugs or valuable contributions to the codebase on [GitHub](https://github.com/nearprotocol/nearcore). Just keep an eye for all the “good first issue” posts. In the meantime, join the channel `#community-validator-announcement` on our [Official Slack](https://near.chat) to be constantly updated, and be the first to know if we plan to offer incentives in the future.
```
Comment (bowenwang1996) [inline docs/validator/validator-faq.md:71]: same here

File: docs/validator/validator-faq.md (hunk 6 above)
Comment (jimmy3dita) [reply docs/validator/validator-faq.md:57]: done

File: website/sidebars.json [hunk 8]
```diff
@@ -170,6 +173,9 @@
           "api/rpc-experimental"
         ]
       }
+    ],
+    "FAQ": [
+      "api/faq"
```
Comment (amgando) [inline website/sidebars.json:178]: i don't think this is what you need for the FAQ file added to validators but i can fix later if you need help with it, no problem.

**Comment by amgando** [comment]
@jimmy3dita I just updated the branch by rebasing off master, resolved some merge conflicts and made a few edits to keeps names consistent (ie. `TestNet` vs TestNet)

**Pull request merged**